Lombiq.ContentEditors/Assets/Scripts/async-editor/async-editor.js

```diff
@@ -0,0 +1,223 @@
+                    nextEditorGroup,
+                    new FormData(self.$refs.editorForm),
+                    (success, data) => { self.processApiData(success, data); });
+            }
+        },
+        processApiData(success, data) {
+            const self = this;
+
+            if (success) {
+                const shouldUpdateQuery = self.contentId !== data.contentId || self.editorGroup !== data.editorGroup;
+
+                self.errorText = '';
+                self.validationSummaryHtml = data.validationSummaryHtml;
+                self.contentId = data.contentId;
+                self.editorHtml = data.editorHtml;
+                self.editorGroup = data.editorGroup;
+                self.editorGroups = data.editorGroups;
+                self.scriptsHtml = data.scriptsHtml;
+                self.message = data.message;
+
+                if (shouldUpdateQuery) self.updateQuery();
+            }
+            else {
+                self.errorText = self.defaultErrorText;
+            }
+        },
+        updateQuery() {
+            const self = this;
+
+            const query = { ...self.$route.query };
+            query[self.asyncEditorId + '.contentId'] = self.contentId;
+            query[self.asyncEditorId + '.editorGroup'] = self.editorGroup;
+            router.push({ path: '/', query: query });
+        },
+        processQuery() {
+            const self = this;
+
+            let shouldLoadEditor = false;
+
+            const contentIdKey = self.asyncEditorId + '.contentId';
+            if (!!Object.getOwnPropertyDescriptor(self.$route.query, contentIdKey) &&
+                self.$route.query[contentIdKey] !== self.contentId) {
+                self.contentId = self.$route.query[contentIdKey];
+                shouldLoadEditor = true;
+            }
+
+            const editorGroupKey = self.asyncEditorId + '.editorGroup';
+            if (!!Object.getOwnPropertyDescriptor(self.$route.query, editorGroupKey) &&
+                self.$route.query[editorGroupKey] !== self.editorGroup) {
+                self.editorGroup = self.$route.query[editorGroupKey];
+                shouldLoadEditor = true;
+            }
+
+            if (shouldLoadEditor) {
+                self.loadEditor();
+            }
+
+            return shouldLoadEditor;
+        },
+        isCurrentGroup(editorGroup) {
+            return editorGroup === this.editorGroup;
+        },
+        isLastGroup(editorGroup) {
+            const self = this;
+
+            return self.editorGroups.at(-1)?.name === (editorGroup ?? self.editorGroup);
+        },
+        getNextEditor(editorGroup) {
+            const editorGroups = this.editorGroups.map((group) => group.name);
+            const index = editorGroups.indexOf(editorGroup ?? this.editorGroup);
+            return editorGroups[index + 1];
+        },
+    },
+};
+
+window.initAsyncEditor = (asyncEditorId, parameters) => {
+    if (!parameters) return;

```

Comment:
How about this? 
```suggestion
    if (!parameters) return window.asyncEditor.editors[asyncEditorId];
```
If the `asyncEditorId` was previously inited it becomes a getter, otherwise it returns `undefined`, just like before.